src/main/java/net/weavemc/loader/api/command/CommandBus.java

```diff
@@ -1,18 +1,26 @@
 package net.weavemc.loader.api.command;
 
+import lombok.experimental.UtilityClass;
 import net.weavemc.loader.api.event.ChatSentEvent;
 import net.weavemc.loader.api.event.EventBus;
-import lombok.experimental.UtilityClass;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Iterator;
 import java.util.List;
 
+/**
+ * The Command Bus manages commands by Weave mods.
+ */
 @UtilityClass
 public class CommandBus {
     private final List<Command> commands = new ArrayList<>();
 
+    /**
+     * While commands can be registered at any time and still work, outside rare cases,
+     * you should be fine registering them whenever you initialize your mod.
```

Comment:
This should be worded a little better, as it insinuates the user is in charge of when their mod is initialized. Maybe registered in your ModInitializer implementation